src/app/client/src/app/modules/content-search/components/view-all/view-all.component.spec.ts

```diff
@@ -199,7 +199,11 @@ describe('ViewAllComponent', () => {
   it('should process the data if view-all is clicked from My-Courses section', () => {
     const courseService = TestBed.get(CoursesService);
     component.sectionName = 'My courses';
-    const sortedData = _.orderBy(_.get(Response, 'enrolledCourseData.enrolledCourses'), ['enrolledDate'], ['desc']);
+    const sortedData = _.map(_.orderBy(_.get(Response, 'enrolledCourseData.enrolledCourses'), ['enrolledDate'], ['desc']), (val) => {
```

Comment:
Why it is different than component.ts code?

Reply:
no sir. both logic of sorting is same.